react-frontend/src/components/LowTouchSaas/collapsedMenus.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import {
+  CollapseButton,
+  CollapseStyled,
+  PanelStyledNoMargin,
+} from '../StyleComponents/collapseMenu';
+import { StyleRichText } from '../StyleComponents/styledMarkdown';
+
+import Planning from './planning';
+import Selection from './selection';
+import PrivacyReview from './privacyReview';
+import SecurityReview from './securityReview';
+import LegalReview from './legalReview';
+import RiskManagementReview from './riskManagementReview';
+import ProcurementAndImplementation from './procurementAndImplementation';
+
+function CollapsedMenus({
+  planning,
```

Comment:
These arguments don't seem to do anything

Reply:
I honestly don't understand the code well enough here to know what could/should be removed. When I played around, I could not get the accordions to expand/collapse if I removed or changed code. 

The ideal would be _not_ to have the first accordion expanded by default (have them all closed by default) -- but it's not the end of the world.

Thank you for the review! I removed all the unused variables I could see here: https://github.com/bcgov/digital.gov.bc.ca/pull/980/commits/e587df903018476bd48e1d3edb53988b565b2a0e

Reply:
I deleted them